Continuously prune the default config file `explcheck-config.toml` (#78)
Witiko/expltools: CHANGES.md

### explcheck v0.10.0

#### Continuous integration

> - Continuously prune file `explcheck-config.toml`. (#78)

Author: ```suggestion
- Continuously prune the default configuration file `explcheck-config.toml`. (#78)
```

>   The default configuration file `explcheck-config.toml` preconfigures many

Author: ```suggestion
  The default config file `explcheck-config.toml` preconfigures many
```

  packages to prevent false positive detections of issues. However, as the
  capabilities of explcheck grow, many of these configurations are outdated
  and no longer necessary.

  This change adds a script `prune-explcheck-config.lua` that reads the default
  configuration and regression test results and then tests which parts of the
  configuration can be removed without affecting the results of the static

>   analysis. This allows the maintainer to remove the outdated configurations.

Author: The script `prune-explcheck-config.sh` should be documented here.

>   analysis. This allows the maintainer to remove the outdated configurations.

Author: ```suggestion
  analysis. This reminds the maintainer to remove the outdated configurations.
```

### explcheck v0.10.0

#### Continuous integration

- Continuously prune the default config file `explcheck-config.toml`. (#78)

  The default config file `explcheck-config.toml` preconfigures many
  packages to prevent false positive detections of issues. However, as the
  capabilities of explcheck grow, many of these configurations are outdated
  and no longer necessary.

  This change adds a script `prune-explcheck-config.lua` that reads the default
  configuration and regression test results and then tests which parts of the
  configuration can be removed without affecting the results of the static

>   analysis. This reminds the maintainer to remove the outdated configurations.

Author: ```suggestion
  analysis. Then, the script reminds the maintainer to remove these parts.
```
